### ISSUE TITLE
Loss function

### ISSUE THREAD

**Issue by Shuhul24** [magicleap/SuperGluePretrainedNetwork#146]
If I am supposed to train the SuperGlue model on my own custom dataset, what should be the loss function? Where is it employed or integrated in the code?

**Comment by sarlinpe** [comment]
You can now use [LightGlue](https://github.com/cvg/LightGlue), we have training code for it.

**Comment by Shuhul24** [comment]
Thanks a lot @sarlinpe. I appreciate your efforts, but for now I want to understand how to train this model (SuperGlue) itself to understand the functioning of the model. Can you please let me know where can i find that or do you got any sample python script example for the same? 
Hope to hear from you
